bigbluebutton-html5/imports/ui/components/video-provider/stream-sorting.ts

```diff
@@ -6,13 +6,13 @@ import { VIDEO_TYPES } from './enums';
 
 const DEFAULT_SORTING_MODE = 'LOCAL_ALPHABETICAL';
 
-// connecting last -> pin first
+// connecting first -> pin first
```

Comment:
No specific reason except that we have to narrow the typying by "eliminating" the `connecting` type. That's because it has a different structure.

I think we should `return 0`, that way we remove the sorting behavior while keeping the typescript requirement.